.gitignore

```diff
@@ -10,3 +10,5 @@ build
 scala-doc-env.sh
 *.swp
 /.conda-env
+.metals
+vivado*
```

Comment:
gitignoring everything starting with vivado in the whole repo doesn't seem like a good idea